src/k8s/pkg/utils/time.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// SecondsToExpirationDate returns the expiration date from now plus the given
+// number of seconds.
+func SecondsToExpirationDate(now time.Time, seconds int) time.Time {
+	switch {
+	case seconds%(365*24*60*60) == 0:
+		return now.AddDate(seconds/365/24/60/60, 0, 0)
+	case seconds%(30*24*60*60) == 0:
+		return now.AddDate(0, seconds/30/24/60/60, 0)
+	case seconds%(24*60*60) == 0:
+		return now.AddDate(0, 0, seconds/24/60/60)
+	default:
+		return now.Add(time.Duration(seconds) * time.Second)
+	}
+}
```

Comment:
## Note
The reasoning behind this change is to standardize our internal definition of a year. While it may not reflect the exact length of a year, the time package will handle the adjustments when we add years, months, or days. This should be passed from the `CLI --> API endpoint --> PKI`